test/inductor/test_aot_inductor.py

```diff
@@ -61,7 +61,7 @@ def load(cls, model, example_inputs, options=None, constraints=None):
             cpp_sources=[launcher],
             functions=["run"],
             extra_ldflags=[so_path],
-            with_cuda=True,
+            with_cuda=True,  # TODO: change this to not is_cpu
```

Comment:
@chenyang78 , you will need to change this to test your runtime logic separation.